tests/Application/config/packages/doctrine_migrations.yaml

```diff
@@ -1,5 +1,4 @@
 doctrine_migrations:
-    dir_name: "%kernel.project_dir%/src/Migrations"
-
-    # Namespace is arbitrary but should be different from App\Migrations as migrations classes should NOT be autoloaded
-    namespace: DoctrineMigrations
+    storage:
+        table_storage:
+            table_name: sylius_migrations
```

Comment:
Is not identical to the plugin skeleton.

Reply:
Could you please tell me where is the difference? I don't see any. I copied the entire file from
https://github.com/Sylius/PluginSkeleton/blob/1.9/tests/Application/config/packages/doctrine_migrations.yaml